src/lib/isMobilePhone.js

```diff
@@ -131,6 +132,7 @@ const phones = {
   'pt-BR': /^((\+?55\ ?[1-9]{2}\ ?)|(\+?55\ ?\([1-9]{2}\)\ ?)|(0[1-9]{2}\ ?)|(\([1-9]{2}\)\ ?)|([1-9]{2}\ ?))((\d{4}\-?\d{4})|(9[1-9]{1}\d{3}\-?\d{4}))$/,
   'pt-PT': /^(\+?351)?9[1236]\d{7}$/,
   'pt-AO': /^(\+244)\d{9}$/,
+  'ro-MD': /^(\+?373|0)((6(0|1|2|6|7|8|9))|(7(6|7|8|9)))\d{6}$/,
```

Comment:
```suggestion
  'ro-MD': /^(\+?373|0)((6([089]\d{6}|(1[01]|2[01]|7[1-7])\d{5}))|(7((6[07]|8[0-8])\d{5}|9\d{6})))$/
```
According to https://www.itu.int/dms_pub/itu-t/oth/02/02/T020200008C0003PDFE.pdf
The result is correct but maybe we RegExp might be simplified a bit so I'm open for suggestions on that

Reply:
We can fix this in a later PR as well

Reply:
I totally agree... we can do a subsequent fix for that.